app/views/layouts/_google_analytics_script.haml

```diff
@@ -1,17 +1,17 @@
 - if (APP_CONFIG.use_google_analytics)
   <script type="text/javascript">
+  - if APP_CONFIG.use_google_tag_manager
+    = "var gtm_key = '#{APP_CONFIG.google_tag_manager_key}';"
 
-  - if feature_enabled?(:universal_analytics)
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    <!-- Google Tag Manager -->
+    (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer',gtm_key);
```

Comment:
I guess we don't need to `gtm_key` variable elsewhere, so you could also replace the variable with `'#{APP_CONFIG.google_tag_manager_key}'`. The `gtm_key` seems to be a global variable attached straight to `window` so removing the variable would pollute the global namespace a bit less.